src/java/us/kbase/mobu/ModuleBuilder.java

```diff
@@ -137,7 +143,9 @@ public static void main(String[] args) throws Exception {
 	        returnCode = runInstallCommand(installArgs, jc);
 	    } else if (jc.getParsedCommand().equals(RUN_COMMAND)) {
 	        returnCode = runRunCommand(runArgs, jc);
-	    }
+	    } else if (jc.getParsedCommand().equals(SHELL_COMMAND)) {
```

Comment:
There's some inconsistent indenting going on here

Reply:
Unfortunately there is a big mix of tabs and spaces throughout ModuleBuilder.java. I changed it to match the "formatting" of the rest of this section (ie. a tab character and then spaces afterward).

eg. line 151 is `<tab-character><four-spaces><tab-character>System.exit(returnCode);`

:man_shrugging: